Doc/library/os.path.rst

```diff
@@ -306,10 +306,10 @@ the :mod:`glob` module.)
 
 .. function:: normcase(path)
 
-   Normalize the case of a pathname.  On Unix and Mac OS X, this returns the
-   path unchanged; on case-insensitive filesystems, it converts the path to
-   lowercase.  On Windows, it also converts forward slashes to backward slashes.
-   Raise a TypeError if the type of *path* is not ``str`` or ``bytes`` (directly
+   Normalize the case of a pathname.  On Windows, converts all characters in the
+   pathname to lowercase, and also converts forward slashes to backward slashes.
+   On other operating systems, return the path unchanged.
+   Raise a :exc:`TypeError` if the type of *path* is neither ``str`` nor ``bytes`` (directly
```

Comment:
In my opinion changing to neither/nor here doesn’t improve the text.

Reply:
@merwok   I think neither/nor are more explicit and formal, see https://english.stackexchange.com/a/10061/44497

Reply:
Yes, but here I think the formalness makes the text heavier, less nice.